scalardb-analytics-spark-sample/sample-data-loader/build.gradle.kts

```diff
@@ -9,10 +9,11 @@ repositories {
 }
 
 dependencies {
-    implementation("com.scalar-labs:scalardb:3.12.1")
-    implementation("com.scalar-labs:scalardb-schema-loader:3.12.1")
+    implementation("com.scalar-labs:scalardb:3.14.0")
+    implementation("com.scalar-labs:scalardb-schema-loader:3.14.0")
     implementation("org.apache.commons:commons-csv:1.10.0")
 
+
```

Comment:
Maybe this empty line is unnecessary?

Reply:
Right. I'll remove it. Thanks!

Reply:
Addresses by https://github.com/scalar-labs/scalardb-samples/pull/75/commits/100cb459f3d6cd5d22199aadc428707a9f18a68b